hub/config/src/main/java/uk/gov/ida/hub/config/domain/X509Certificate.java

```diff
@@ -0,0 +1,30 @@
+package uk.gov.ida.hub.config.domain;
+
+import com.fasterxml.jackson.annotation.JsonAlias;
+import com.fasterxml.jackson.annotation.JsonCreator;
+import com.fasterxml.jackson.annotation.JsonIgnore;
+import com.fasterxml.jackson.annotation.JsonProperty;
+
+import static java.text.MessageFormat.format;
+import static uk.gov.ida.common.shared.security.Certificate.BEGIN_CERT;
+import static uk.gov.ida.common.shared.security.Certificate.END_CERT;
+
+/* This class is modified from X509CertificateConfiguration in security-utils due to an
+* outstanding bug https://github.com/FasterXML/jackson-databind/issues/1358 which means
+* we cannot directly deserialize to X509CertificateConfiguration */
+public class X509Certificate {
+    @JsonProperty
```

Comment:
The AppRule base class serializes the DTOs that use this so it needs some JsonProperty annotations